front-end/src/components/Post/Poll/GeneralSignals.tsx

```diff
@@ -82,26 +81,23 @@ const CouncilSignals = ({ className, ayes, ownVote, nays, postId, refetch }: Pro
 				{error?.message && <FilteredError className='info' text={error.message}/>}
 			</div>
 			{id && (ownVote
-				? <>
-					<div className='info text-muted'>You voted {ownVote}.</div>
-					<Form standalone={false}>
-						<Form.Group>
-							<Form.Field>
-								<Button
-									fluid
-									basic
-									className='primary negative'
-									onClick={cancelVote}
-								>
-									Cancel Vote
-								</Button>
-							</Form.Field>
-						</Form.Group>
-					</Form>
-				</>
+				?
```

Comment:
We're destroying the `AyeNayButtons` here, it's becoming a monster. 
1) I'm sure we can forget about the `hasVoted` props, and handle this styling in the `GeneralSignals` component, rather than in the `AyeNayButtons`.
2) I'm pretty sure there is no need for this condition on ownVote ? ... : ... if you make things like:
```js
				<AyeNayButtons
						disabled={!!ownVote}
						className='signal-btns'
						onClickAye={() => castVote(Vote.AYE)}
						onClickNay={() => castVote(Vote.NAY)}
					/>
```

And then depending on the vote, you style the disabled buttons. If they are disabled, the function will not work either (which is what we want). This removes the condition and alls the super specific props that where added.